app/models/mixins/relationship_mixin.rb

```diff
@@ -54,6 +55,22 @@ def clear_relationships_cache(*args)
     @association_cache.delete(:all_relationships)
   end
 
+  # this allows us to call similarly named ancestry or mixin methods
+  # depending on whether default_relationship_type is set to :genealogy or not
+
+  def call_ancestry_method(method_name, *args)
+    bound_method = ((@@ancestry_method ||= {})[method_name] ||= ::Ancestry::InstanceMethods.instance_method(method_name)).bind(self)
+    if !args.first.nil? && args.flatten != [] && args.flatten != [{}] # ? !args.empty?
+      bound_method.call(*args)
+    else
+      bound_method.call
+    end
+  end
+
+  def use_ancestry?
+    relationship_type == :genealogy
```

Comment:
I guess I'm commenting more on this line?  I assumed this wouldn't be needed or would be derivable?

Reply:
@Fryguy I'm trying better to understand your comment here.

do you have suggestions on how we would derive the types of relationships we want to use for ancestry and the ones we want to use for relationships?

Reply:
@d-m-u put in a variable here that we can set in the class.

That way this will be quick enough and allow us to refactor and slowly migrate away from relationships when it makes sense.

I think copying the existing class variable makes sense. 
Would like the developer to add to the array instead of setting it

something like
```ruby
cattr_accessor :default_relationship_type
cattr_accessor :skip_relationships

def skip_relationships_for(relationship_name)
  self.skip_relationships ||= ...
end
```

Reply:
I thin I was expecting the whole thing in one PR, but if you're breaking it up into pieces, then I can understand this line.  Ultimately I was expecting that this line wouldn't have to exist when everything is done.